spec/spec_helper.rb

```diff
@@ -4,6 +4,7 @@
   require 'curb'
   require 'patron'
   require 'typhoeus'
+  require 'em-http'
```

Comment:
Ah, that’s my two cents. Can you reorder it as follows?

```suggestion
  require 'curb'
  require 'em-http'
  require 'patron'
  require 'typhoeus'
```